lang/en/local_linkchecker_robot.php

```diff
@@ -81,7 +81,7 @@
 $string['size'] = 'Size';
 $string['url'] = 'URL';
 $string['mimetype'] = 'Mime type';
-$string['broken'] = 'Broken URL';
```

Comment:
also this should be something like 'Broken Links / URLS' - I think you did this and the commits are mixed up somewhere